packages/babel-cli/src/babel/file.js

```diff
@@ -184,15 +199,14 @@ export default async function({ cliOptions, babelOptions }) {
           }
 
           console.error(err);
-          return null;
```

Comment:
I would suggest leave `return null` here so that we can make this PR focus on adding flow type annotation _without_ changing any internal logic.

Note that you can use union type to specify this function returns either a `Promise<Object>` or `null`.
```js
_filenames.map(async function(filename: string): Promise<Object> | null {
```

Reply:
Good point. I'll make the fix.